libsql-server/src/namespace/configurator/helpers.rs

```diff
@@ -328,8 +329,31 @@ where
             line = tokio::task::spawn_blocking({
                 let conn = conn.clone();
                 move || -> crate::Result<String, LoadDumpError> {
-                    conn.with_raw(|conn| conn.execute(&line, ())).map_err(|e| {
-                        LoadDumpError::Internal(format!("line: {}, error: {}", line_id, e))
+                    conn.with_raw(|conn| {
+                        conn.authorizer(Some(|auth: AuthContext<'_>| match auth.action {
```

Comment:
I want to say we don't re-use libsql connections unless a user does it via hrana etc. So its controlled by the user of the connection api rather than internally.

Reply:
Ok, just wanted to clarify this as we install authorizer and if connection will be reused - user will be unable to `ATTACH` db (which may be valid action)